eglot-jl.el

```diff
@@ -90,5 +116,20 @@ Otherwise returns nil"
                ;; function instead of strings to find project dir at runtime
                '(julia-mode . eglot-jl--ls-invocation)))
 
+;;;###autoload
+(defun eglot-jl-compile-sysimage ()
+  "Create a compiled system image to reduce subsequent language
```

Comment:
Please fit a single sentence description on the first line of docstrings.

Reply:
This should be done in the latest version.